specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1235 @@
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "freshness",
+            "x-ms-client-name": "Freshness",
+            "in": "query",
+            "description": "Filter search results by the following age values: Day—Return webpages that Bing discovered within the last 24 hours. Week—Return webpages that Bing discovered within the last 7 days. Month—Return webpages that discovered within the last 30 days. This filter applies only to webpage results and not to the other results such as news and images.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Day",
+              "Week",
+              "Month"
+            ],
+            "x-ms-enum": {
+              "name": "Freshness",
+              "modelAsString": false
+            }
+          },
+          {
+            "name": "mkt",
+            "x-ms-client-name": "Market",
+            "in": "query",
+            "description": "The market where the results come from. Typically, mkt is the country where the user is making the request from. However, it could be a different country if the user is not located in a country where Bing delivers results. The market must be in the form <language code>-<country code>. For example, en-US. The string is case insensitive. If known, you are encouraged to always specify the market. Specifying the market helps Bing route the request and return an appropriate and optimal response. If you specify a market that is not listed in Market Codes, Bing uses a best fit market code based on an internal mapping that is subject to change. This parameter and the cc query parameter are mutually exclusive—do not specify both.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "offset",
+            "x-ms-client-name": "Offset",
+            "in": "query",
+            "description": "The zero-based offset that indicates the number of search results to skip before returning results. The default is 0. The offset should be less than (totalEstimatedMatches - count). Use this parameter along with the count parameter to page results. For example, if your user interface displays 10 search results per page, set count to 10 and offset to 0 to get the first page of results. For each subsequent page, increment offset by 10 (for example, 0, 10, 20). it is possible for multiple pages to include some overlap in results.",
```

Comment:
perhaps add a `default` and  `minimum` and `maximum` values?
``` json 
"default" : 10,
"minimum": 1,
"maximum":50 
```

Reply:
Tracked